ui/components/MeshConfigComponent.js

```diff
@@ -711,12 +634,20 @@ handleNATSClick = () => {
           </ListItem>
         </List>
       );
-      if (configuredServer) {
-        chp = <Tooltip title={`Server: ${configuredServer}`}>{chp}</Tooltip>;
-      }
       showConfigured = (
         <div>
-          {chp}
+          {contexts.map(ctx => (
```

Comment:
we are essentially pinging the kubernetes server right?, all contexts will have the same kubernetes server right. In that case, don't you think we should just have a single chip ? doesn't having multiple chips each with the labels of contexts give us a false idea that the ping is associated with the context ? 
// @leecalcote

Reply:
Each kubernetes context is associated with its own kube config and it's own kubernetes server. So essentially, we need multiple chips, one for each Cluster.

Reply:
Thank you for the clarification @Revolyssup 🙏